### PR TITLE
[WIP] Sticky Page Tabs

### PR DESCRIPTION
**Pull request by lovemecomputer**
WIP sticky tab implementation of https://github.com/carbon-design-system/carbon-website/issues/578

### CODE REVIEW

File: packages/addons-website/scss/components/_website-header.scss [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 
 .#{$prefix}--header--website {
-  z-index: 9999;
+  z-index: 13000;
   background-color: $ui-05;
   border-bottom: 1px solid $ibm-colors__gray-80;
+  transition: 0.35s cubic-bezier(0.2, 0, 0.38, 0.9) top;
+}
+
+.sticky--tabs.sticky-tabs--header-hidden .#{$prefix}--header--website {
```
Comment (alisonjoseph) [inline packages/addons-website/scss/components/_website-header.scss]: Lets keep this css separate from the header add-on for now. Maybe put everything related to this in a sticky-tabs scss file.

File: packages/addons-website/scss/components/_website-header.scss [hunk 2]
```diff
@@ -1,9 +1,14 @@
 @import '../prefix';
 
 .#{$prefix}--header--website {
-  z-index: 9999;
+  z-index: 13000;
   background-color: $ui-05;
   border-bottom: 1px solid $ibm-colors__gray-80;
+  transition: 0.35s cubic-bezier(0.2, 0, 0.38, 0.9) top;
```
Comment (alisonjoseph) [inline packages/addons-website/scss/components/_website-header.scss]: did these transitions come from Shixie? I think there should be a variable for motion stuff.

File: packages/addons-website/scss/components/_website-header.scss (hunk 1 above)
Comment (lovemecomputer) [reply packages/addons-website/scss/components/_website-header.scss]: @alisonjoseph will do!!

File: src/components/Layouts/index.js [hunk 3]
```diff
@@ -31,10 +31,44 @@ class Layout extends React.Component {
...
+      body.scrollHeight,
+      body.offsetHeight,
+      html.clientHeight,
+      html.scrollHeight,
+      html.offsetHeight
+    );
+
+    var value = document.body.scrollTop;
+
+    console.log(`docHeight = ${docHeight}`);
```
[the remark this answers is not in the document]
Comment (lovemecomputer) [reply src/components/Layouts/index.js]: @asudoh yeah, thanks for checking on it tho! i'm still sorting out logic for the 3 different states : )

File: packages/addons-website/scss/components/_website-header.scss (hunk 2 above)
Comment (lovemecomputer) [reply packages/addons-website/scss/components/_website-header.scss]: @alisonjoseph thanks! the motion content page listed this equation but not a token, so i just copied the equation. i'll check for the token before removing [WIP]!

**Comment by lovemecomputer** [comment]
old PR, closed in favor of https://github.com/carbon-design-system/carbon-website/pull/1453

**Pull request closed without merging**